src/index.html

```diff
@@ -5,9 +5,72 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card body__card" data-qa="card">
+      <div class="card__picture">
```

Comment:
1. Think of using `<article>` here 
<img width="398" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/19c21f09-3882-43fa-ac8b-d05a289884d0">
2. No need in `.card__picture`wrapper